### PR TITLE
bump version to 3.0.0, update release notes

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

File: RELEASE_NOTES [hunk 1]
```diff
@@ -3,7 +3,45 @@ For more detailed information, please see the git log.
...
+  - add support for giving /lib preference over /lib64 & co in GCC installation (#1030, #1035)
+  - enable installation of libliberty by default for binutils (#1030)
+  - avoid CMake fiddling with the RPATHs injected by EasyBuild via --rpath in CMakeMake and METIS easyblocks (#1031, #1034)
+  - simplify scipy sanity check to make it more robust w.r.t. version updates (#1037)
+- various bug fixes, including:
+  - make sure 'None' doesn't appear in modules generated with --module-only (#998)
+  - fix ATLAS easyblock for non-x86 systems (#1003)
+  - fix 'usempi' and 'with_mpi' usage to allow for a serial build of Amber 16 (#1013)
+  - add both lib/python2.7/site-packages/{,wx-3.0-gtk2} to $PYTHONPATH for wxPython (#1018)
+  - only hard inject RPATH for /usr/lib* directories when building bintuils with dummy toolchain (#1026)
```
Comment (fgeorgatos) [inline RELEASE_NOTES]: binutils

File: RELEASE_NOTES [hunk 2]
```diff
@@ -3,7 +3,45 @@ For more detailed information, please see the git log.
...
+  - update QuantumESPRESSO easyblock to support packaging changes in 6.0 (#1007)
+  - add support to Scons generic easyblock to provide argument to specify installation prefix (#1008)
+  - update IntelBase and imkl easyblocks to handle the 2017 versions of compilers/imkl (#1012)
+  - leverage toolchain.linalg functionality in ScaLAPACK easyblock, use copy_file (#1014)
+  - allow netCDF-C++4 to be used with ESMF (#1019)
+  - update Advisor easyblock to support latest versions (#1021)
+  - update CBLAS easyblock to build with foss toolchain (#1024)
+  - update Gurobi easyblock to use copy_file (#1028)
+  - add support for giving /lib preference over /lib64 & co in GCC installation (#1030, #1035)
+  - enable installation of libliberty by default for binutils (#1030)
```
Comment (fgeorgatos) [inline RELEASE_NOTES]: libiberty is proper name? I guess people who care about it wouldn't be too much surprised though

File: RELEASE_NOTES [hunk 3]
```diff
@@ -3,7 +3,45 @@ For more detailed information, please see the git log.
...
+  - remove QLogicMPI easyblock (#1023)
+- new easyblock for installing Anaconda (#950)
+- add generic easyblock for Conda installations (#950)
+- various enhancements, including:
+  - enable use of GCCcore as toolchain for Clang, fail if no GCC prefix is found (#1002)
+  - also build Boost MPI library in parallel (#1005, #1038)
+  - enhance g2clib easyblock to allow to install 1.6.0 or higher (#1006)
+  - update QuantumESPRESSO easyblock to support packaging changes in 6.0 (#1007)
+  - add support to Scons generic easyblock to provide argument to specify installation prefix (#1008)
+  - update IntelBase and imkl easyblocks to handle the 2017 versions of compilers/imkl (#1012)
```
Comment (fgeorgatos) [inline RELEASE_NOTES]: with thanks! 

ok, see other comments, lgtm

**Comment by boegel** [comment]
Thanks for the review @fgeorgatos!

**Pull request merged**